cross/tvheadend/Makefile

```diff
@@ -1,11 +1,11 @@
 PKG_NAME = tvheadend
 PKG_VERS = 4.3
 PKG_EXT = tar.gz
-PKG_DOWNLOAD_METHOD = git
 PKG_GIT_HASH = 9ed76c0a176b055a57b6e8bd2e0b6e29409269a9
-PKG_DIST_SITE = https://github.com/tvheadend/tvheadend.git
-PKG_DIST_FILE = $(PKG_NAME)-git$(PKG_GIT_HASH).$(PKG_EXT)
-PKG_DIR = $(PKG_NAME)-git$(PKG_GIT_HASH)
+PKG_DIST_NAME = $(PKG_GIT_HASH).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/tvheadend/tvheadend/archive
+PKG_DIST_FILE = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIR = $(PKG_NAME)-$(PKG_GIT_HASH)
 
```

Comment:
No no no!!!
github 4.3 package dates from 2017.
`tvheadend` does not provide package version anymore.
We must use a git hash specific snapshot.
As such I'm using the date in conjunction with the git hash to generate a proper version schema.
Please revert this.